infra/aws/e2e/accounts/bulk-create.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+set -x
```

Comment:
You should also have `set -o pipefail` because of piped commands. And also `set -o nounset`.